typescript/packages/plugins/coingecko/README.md

```diff
@@ -0,0 +1,40 @@
+# Goat Coingecko Plugin 🐐 - TypeScript
+
+Coingecko plugin for Goat. Allows you to create tools for interacting with the CoinGecko API.
+
+## Installation
+```
+npm install @goat-sdk/plugin-coingecko
+```
+
+## Usage
+    
+```typescript
+import { coingecko } from "@goat-sdk/plugin-coingecko";
+
+const plugin = coingecko({ key: process.env.COINGECKO_API_KEY as string });
+
+// Get trending coins
+const trendingCoins = await plugin.getTrendingCoins();
```

Comment:
The plugin returns an array so you can't really really call the function like this.

For the readme I would simply add how you get the plugin initialized and describe the actions it has